packages/layout/src/components/StackedAccordionSection/index.tsx

```diff
@@ -0,0 +1,214 @@
+import React, { useCallback, useContext, useEffect, useRef, useState } from 'react';
+import { useUniqueIdentifier } from '@rainbow-modules/hooks';
+import { RenderIf } from 'react-rainbow-components';
+import {
+    StyledSection,
+    StyledSectionContent,
+    StyledSectionHeader,
+    StyledResizeBar,
+} from './styled';
+import { StackedAccordionSectionProps, MoveProps } from './types';
+import context from '../StackedAccordion/context';
+import { Context } from '../StackedAccordion/types';
+import RightArrow from './rightArrow';
+import { useResize } from './hooks/useResize';
+import shouldMove from '../ResizableColumns/helpers/shouldMove';
```

Comment:
if this helper will be reusable by two o more components, then I think we should move it to a helper folder inside `src`